docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -65,6 +66,57 @@ Applications that target versions of .NET Framework starting with 4.7.1 can disa
+- Fixed the color of a `DataGridViewLinkCell` when the cell is selected.
+- Fixed an issue with custom <xref:System.Windows.Forms.DataGridView> controls where there was no `ControlType` or `LocalizedControlType` provided for custom <xref:System.Windows.Forms.DataGridViewCell> elements.
+- Updated the luminosity ratio to 3.5:1 for <xref:System.Windows.Forms.ToolStripButton> controls that have `ToolStripRenderMode` set to `System`.
+- Improved keyboard navigation in a <xref:System.Windows.Forms.ToolStrip> when the element is a `ToolStripComboBox` type.
+- Updated the background color of the <xref:System.Windows.Forms.ToolStripButton> in high contrast mode.
+- Ensured that there is a bounding rectangle reported to assistive technology for the <xref:System.Windows.Forms.ToolStripSeparator>.
+- Fixed an issue that could cause the screen reader JAWS to crash when reading the <xref:System.Windows.Forms.PropertyGrid> control.
+- Ensured that the UIA hierarchy tree for a <xref:System.Windows.Forms.PropertyGrid> control is updated when a complex entry like `Font` is expanded. Also ensure that the tree is updated properly when the entry is then collapsed and no longer visible.
+- <xref:System.Windows.Forms.PropertyGrid> categories now have a localized control type of `PropertyGrid category`.
+- Fixed an issue with the <xref:System.Windows.Forms.ComboBox> that could cause the app to crash under Accessibility Insights for Windows.
+- Updated the border color for the <xref:System.Windows.Forms.Button> to have more contrast in the default colors.
+- Enabled assistive technology tools to access the `ControlBox` buttons of a maximized MDI child form.
+- The `AccessibleName` property of a <xref:System.Windows.Forms.DomainUpDown> control has a new default value of an empty string. The empty string will encourage developers to create a meaningful name rather than accepting the previous default value.
+- Updated the `AccessibleName` property of the Print button in the <xref:System.Windows.Forms.PrintPreviewDialog> from "Print Button" to "Print" to avoid redundancy when screen readers announce the control and its type.
+- Updated UIA list controls to remove an empty list element when a <xref:System.Windows.Forms.PropertyGrid> cell of type `ComboBox` is closed and no longer visible.
+
+<a name="wpf481"></a>
+
+### Windows Presentation Foundation (WPF)
+
+#### Accessible Tooltip handling improvements
+
+In this release, WPF improved the experience by ensuring that a tooltip in the current window can easily be dismissed by using the Esc key, the Ctrl key (by itself), or by the combination Ctrl+Shift+F10. The scope of the Esc key was reduced in this release to apply only to the current window, when previously it would have applied to any open tooltip in the application. By default WPF tooltips will follow [WCAG2.1 guidelines](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) to be persistent, dismissable, and hoverable.
```

Comment:
```suggestion
In this release, WPF improved the experience by ensuring that a tooltip in the current window can easily be dismissed by using the Esc key, the Ctrl key (by itself), or by the combination Ctrl+Shift+F10. The scope of the Esc key was reduced in this release to apply only to the current window, when previously it would have applied to any open tooltip in the application. By default, WPF tooltips will follow [WCAG2.1 guidelines](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) to be persistent, dismissable, and hoverable.
```